### PR TITLE
[RFR] Added “PhantomJS” entry

### PR DESCRIPTION
**Pull request by virtyaluk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -146,7 +146,7 @@ If you are willing to contribute, open a pull request to complete, update or fil
 
 ### P
 
-* [PhantomJS](/glossary/PHANTOMJS.md): *to be completed.*
+* [PhantomJS](/glossary/PHANTOMJS.md): a headless WebKit scriptable with a JavaScript API.
```
Comment (KittyGiraudel) [inline README.md]: I find this sentence, while perfectly accurate, not very easy to understand. Do you think we could make it more within reach?

File: glossary/PHANTOMJS.md [hunk 2]
```diff
@@ -1,3 +1,21 @@
 # PhantomJS
 
-[PhantomJS](http://phantomjs.org/) section to be completed.
+[PhantomJS](http://phantomjs.org/) is a headless WebKit scriptable with a JavaScript API. It has **fast** and **native** support for various web standards: DOM handling, CSS selector, JSON, Canvas, and SVG.
```
Comment (KittyGiraudel) [inline glossary/PHANTOMJS.md]: selectors\* I would say?

File: glossary/PHANTOMJS.md [hunk 3]
```diff
@@ -1,3 +1,21 @@
...
+
+## Use Cases
+
+- **Headless web testing**. Lightning-fast testing without the browser is now possible! Various [test frameworks](http://phantomjs.org/headless-testing.html) such as Jasmine, Capybara, QUnit, Mocha, WebDriver, YUI Test, BusterJS, FuncUnit, Robot Framework, and many others are supported.
```
Comment (KittyGiraudel) [inline glossary/PHANTOMJS.md]: We could link to existing entries here. :)

**Comment by KittyGiraudel** [comment]
Hey, thank you very much! I added a few comments but the only one you should take care of is the first one, about making the short description a bit more accessible. I can deal with all the formatting / typos / cross-linking stuff. :)

**Comment by virtyaluk** [comment]
Got it! I'll send PR with changes a little bit later. Thanks for reviewing!

**Comment by virtyaluk** [comment]
@HugoGiraudel 
Hey, I've committed some changes. So, would you mind to review them?

**Comment by KittyGiraudel** [comment]
Brilliant, thank you!

**Pull request merged**